pkg/cri/store/container/container.go

```diff
@@ -78,6 +81,14 @@ func WithStatus(status Status, root string) Opts {
 	}
 }
 
+// WithOnceSignal only send the signal once.
+func WithOnceSignal() Opts {
```

Comment:
might be overkill to have this be done with a With.. since there's only one initial state.. you can just put this code in the NewContainer() call below